module_1/gradle/by.epam.module1.gradle.utils/src/test/java/StringUtilsTest.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class StringUtilsTest {
+
+    @Test
+    void isTestPositiveNumber() {
+        boolean trueResult = StringUtils.isPositiveNumber("12");
```

Comment:
trueResult is not a good name for the variable.  in this case, you can get true or false here. Just use "result"

Reply:
The same story with all resultResult below